lib/src/annotate.rs

```diff
@@ -358,9 +391,9 @@ fn get_file_contents(
                 })?;
             Ok(file_contents.into())
         }
-        MaterializedTreeValue::FileConflict { contents, .. } => {
-            Ok(materialize_merge_result_to_bytes(&contents))
-        }
+        MaterializedTreeValue::FileConflict { contents, .. } => Ok(
+            materialize_merge_result_to_bytes(&contents, conflict_marker_style),
```

Comment:
nit: just hardcode the default here?

This is just a workaround until we implement a proper tracking of conflict files.

Reply:
Ok, that makes sense to me! I updated it.